packages/server/src/build-hash.ts

```diff
@@ -1,34 +1,52 @@
-import {hashElement} from "folder-hash"
-import {pathExists, readFile, writeFile} from "fs-extra"
+import {hashElement,HashElementNode} from "folder-hash"
+import {pathExists,readFile,writeFile} from "fs-extra"
 import {resolve} from "path"
 
-export async function getInputArtefactsHash() {
+const debug = require("debug")("blitz:build-hash")
+
+export async function getInputArtifacts() {
```

Comment:
Artefact is UK-english, artifiact is US.. generally I guess we should default to US